src/styles/sidebar/styled-text.scss

```diff
@@ -1,4 +1,7 @@
 @mixin styled-text() {
+  // Reset the line-height in case any parent elements have set it.
+  line-height: normal;
```

Comment:
`normal` here means "whatever the browser uses by default", which according to MDN, is a small multiple (eg. 1.2) of _the element's font size_.